updates/35/updates.sql

```diff
@@ -639,4 +639,180 @@ INSERT INTO assets(identifier, name, type) VALUES('SBTC', 'Super Bitcoin', 'W');
+*
+INSERT INTO assets(identifier, name, type) VALUES('LOCKIN', 'LOCK IN', 'Y'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('LOCKIN', 'LOCKIN', 'lock-in', 'LOCKIN', NULL, 1710530648, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0xB1D1eae60EEA9525032a6DCb4c1CE336a1dE71BE', 'Derive', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0xB1D1eae60EEA9525032a6DCb4c1CE336a1dE71BE', 'A', 1, '0xB1D1eae60EEA9525032a6DCb4c1CE336a1dE71BE', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0xB1D1eae60EEA9525032a6DCb4c1CE336a1dE71BE', 'DRV', 'derive', 'DRV', NULL, 1732055471, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:10/erc20:0x33800De7E817A70A694F31476313A7c572BBa100', 'Derive', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:10/erc20:0x33800De7E817A70A694F31476313A7c572BBa100', 'A', 10, '0x33800De7E817A70A694F31476313A7c572BBa100', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:10/erc20:0x33800De7E817A70A694F31476313A7c572BBa100', 'DRV', 'derive', 'DRV', NULL, 1733806403, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:42161/erc20:0x77b7787a09818502305C95d68A2571F090abb135', 'Derive', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:42161/erc20:0x77b7787a09818502305C95d68A2571F090abb135', 'A', 42161, '0x77b7787a09818502305C95d68A2571F090abb135', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:42161/erc20:0x77b7787a09818502305C95d68A2571F090abb135', 'DRV', 'derive', 'DRV', NULL, 1733807283, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:8453/erc20:0x9d0E8f5b25384C7310CB8C6aE32C8fbeb645d083', 'Derive', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:8453/erc20:0x9d0E8f5b25384C7310CB8C6aE32C8fbeb645d083', 'A', 8453, '0x9d0E8f5b25384C7310CB8C6aE32C8fbeb645d083', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:8453/erc20:0x9d0E8f5b25384C7310CB8C6aE32C8fbeb645d083', 'DRV', 'derive', 'DRV', NULL, 1736644573, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('USDG', 'Global Dollar', 'W'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('USDG', 'USDG', 'global-dollar', 'USDG', NULL, 1730851200, NULL);
```

Comment:
This one is an ethereum token https://etherscan.io/token/0xe343167631d89B6Ffc58B88d6b7fB0228795491D